.vscode/settings.json

```diff
@@ -66,11 +66,11 @@
     },
   ],
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
Why are you removing lint fixes?

Reply:
I made it explicit because the initial value was not automatically formatting the code on save.